src/app/app.css

```diff
@@ -1,7 +1,10 @@
 :host {
-  
-  & section {
+  & {
+    min-height: 100vh;
     text-align: center;
+    display: flex;
```

Comment:
Nice, having flexbox in this project is a great for having an example / reference of.  👍